### PR TITLE
Bump apko

### PR DESCRIPTION
**Pull request by jonjohnsonjr**
Plumb remote.Options down into apko.

### CODE REVIEW

File: internal/provider/config_data_source_test.go [hunk 1]
```diff
@@ -34,10 +37,13 @@ data "apko_config" "this" {
 			Config: `
 data "apko_config" "this" {
   config_contents = <<EOF
-  archs:
-  - x86_64
-  - arm64
-  EOF
+contents:
+  repositories:
+  - ./packages
```
Comment (imjasonh) [inline internal/provider/config_data_source_test.go:42]: Why is this added?

Comment (jonjohnsonjr) [reply internal/provider/config_data_source_test.go:42]:
```
--- FAIL: TestAccDataSourceConfig (1.31s)
    config_data_source_test.go:16: Step 1/2 error: Error running pre-apply refresh: exit status 1

        Error: error computing package locks

          with data.apko_config.this,
          on terraform_plugin_test.tf line 2, in data "apko_config" "this":
           2: data "apko_config" "this" {

        initializing apk: failed to initialize apk repositories: must provide at
        least one repository
```

https://github.com/chainguard-dev/go-apk/blob/1bf6ea16cc454e153246afc871d8c9bf0ba577c7/pkg/apk/repo.go#L89

Comment (mattmoor) [reply internal/provider/config_data_source_test.go:42]: I have a different fix for this in: https://github.com/chainguard-dev/terraform-provider-apko/pull/93

Comment (mattmoor) [reply internal/provider/config_data_source_test.go:42]: I'm fine with this tho

**Review by mattmoor — APPROVED** [review]
